beat_gan.ipynb

```
@@ -0,0 +1,988 @@
+    "                X_train.append(normed)\n",
+    "    return np.array(X_train) if policy == 1 else X_train"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def generate_after_training(BATCH_SIZE):\n",
+    "    generator = generator_model()\n",
```

Ruff (0.13.3)

Unused function argument: `X_train`

(ARG001)